src/components/Loading/loading.spec.js

```diff
@@ -3,9 +3,10 @@ import { render, screen } from "@testing-library/react";
 import Loading from "./index";
 
 describe("Loading Component", () => {
-  test("renders loading spinner and text", () => {
+  it("renders loading spinner and text", () => {
     const text = "Loading...";
     render(<Loading text={text} />);
+
     const spinner = screen.getByTestId("spinner");
     const loadingText = screen.getByText(text);
     expect(spinner).toBeInTheDocument();
```

Comment:
```suggestion
    const loadingText = screen.getByText(text);
    
    expect(spinner).toBeInTheDocument();
```
`expect` 인 결과 확인문과는 개행을 해주는 것이 맞아보입니다!